gateway-service-impl/src/main/java/org/hypertrace/gateway/service/explore/ExploreService.java

```diff
@@ -124,16 +121,19 @@ private IRequestHandler getRequestHandler(
               request.getTimeAggregationList(),
               request.getOrderByList(),
               request.getGroupByList());
+
+      boolean allFieldsOnEDS =
```

Comment:
Okay so here is my confusion, if there is request.getIncludeNonLiveEntities(), then all are needed to be on EDS not as the only option. Say all fields are on QS, EDS, then by default QS would be selected because it will skip all the checks.  

Thats why needed these two checks and that's why field is named allFieldsOnEDS